### PR TITLE
Updated world.md to use const + some code styling

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request updates `world.md` to use `const` instead of `var` and adds some code styling**

### CODE REVIEW

**Comment by Marketionist** [comment]
@aslakhellesoy if it's possible I would be glad to join Cucumber committers team to add more docs and code updates

**Comment by Marketionist** [comment]
@charlierudolph @aslakhellesoy please review

**Comment by Marketionist** [comment]
@aslakhellesoy please review and if it's possible I would be glad to join Cucumber committers team to add more docs and code updates

File: docs/support_files/world.md [hunk 1]
```diff
@@ -4,20 +4,20 @@
 The default world constructor is:
 
 ```javascript
-function World({attach, parameters}) {
-  this.attach = attach
-  this.parameters = parameters
+function World({ attach, parameters }) {
+  this.attach = attach;
```
Comment (aslakhellesoy) [inline docs/support_files/world.md]: Please remove the semicolons. We don't use them in the codebase, so I'd prefer to be consistent in code examples.

Comment (Marketionist) [reply docs/support_files/world.md]: Done

**Comment by aslakhellesoy** [comment]
Hi @Marketionist,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**